libs/SalesforceSDKCore/SalesforceSDKCore/SalesforceSDKCore.h

```diff
@@ -2,7 +2,7 @@
  SalesforceSDKCore.h
  SalesforceSDKCore
 
- Created by Bharath Hariharan on Fri Feb  9 09:53:54 PST 2018.
+ Created by Raj Rao on Thu Feb 22 15:09:12 PST 2018.
```

Comment:
@trooper2013 Why didn't the new public header get added here? Could you wipe `DerivedData` and build the `Everything` scheme?